functions/__sf_section_dir.fish

```diff
@@ -14,6 +14,11 @@ function __sf_section_dir -d "Display the current truncated directory"
 	__sf_util_set_default SPACEFISH_DIR_TRUNC_REPO true
 	__sf_util_set_default SPACEFISH_DIR_COLOR cyan
 
+	# Write Permissions lock symbol
+	__sf_util_set_default SPACEFISH_DIR_LOCK_SHOW true
+	__sf_util_set_default SPACEFISH_DIR_LOCK_SYMBOL "🔒"
```

Comment:
It appears spaceship uses the powerline font's lock as opposed to the emoji lock `" "`
I would suggest doing the same to maintain consistency.
https://github.com/denysdovhan/spaceship-prompt/blob/0f4f6840ffba60c1f8b3e329ff2ee5a69d48a983/sections/dir.zsh#L17

Reply:
I used Powerline lock symbol from the Character Viewer app in mac. Didn't work.
Now I've just copied from spaceship. It works.